qiita_pet/templates/list_studies.html

```diff
@@ -58,6 +58,7 @@
   $('#user-studies-table').dataTable({
       "lengthMenu": [[5, 10, 50, -1], [5, 10, 50, "All"]],
       "deferRender": true,
+      "sDom": '<"top">rti<"bottom"p>',
```

Comment:
I don't understand this `<"bottom"p>` syntax. but if this is ok, then the rest looks fine

Reply:
The syntax comes from: http://legacy.datatables.net/usage/options, but the screenshot shows the differences.